protocol/x/revshare/keeper/revshare.go

```diff
@@ -124,18 +124,18 @@ func (k Keeper) SetUnconditionalRevShareConfigParams(ctx sdk.Context, config typ
 	store.Set([]byte(types.UnconditionalRevShareConfigKey), unconditionalRevShareConfigBytes)
 }
 
-// ValidateRevShareSafety roughly checks if the total rev share is valid using the formula below
-// highest_affiliate_taker_share * (highest_taker_fee_share_ppm /
-// (lowest_maker_fee_share_ppm + highest_taker_fee_share_ppm))
+// ValidateRevShareSafety roughly checks if the total rev share is valid using the formula below:
+// highest_affiliate_taker_share * (lowest_taker_fee_ppm /
+// (lowest_maker_ppm + lowest_taker_fee_ppm))
 // + sum(unconditional_rev_shares) + market_mapper_rev_share < 100%
 func (k Keeper) ValidateRevShareSafety(
```

Comment:
Since both `totalUnconditionalRevSharePpm` and `totalMarketMapperRevSharePpm` are post maker rebates and taker affiliate rev share, I think we just need:
- totalUnconditionalRevSharePpm + totalMarketMapperRevSharePpm < 100
- Highest_affiliate_rev_share[protocol constant of 50%] * lowest_taker_fee - lowest_maker_fee < lowest_taker_fee

Which is actually a lot simpler 😓 Lmk if this makes sense

Reply:
This makes sense to me. Made the change!

Reply:
Discussed in thread, but let's update equation to most human-readable form: 
```
Highest_affiliate_rev_share[protocol constant of 50%] * lowest_taker_fee < lowest_taker_fee + lowest_maker_fee`
```